framework-docs/modules/ROOT/pages/testing/mockmvc/assertj/assertions.adoc

```diff
@@ -47,3 +47,16 @@ include-code::./FamilyControllerTests[tag=assert-file,indent=0]
 
 If you prefer to use another library, you can provide an implementation of
 {spring-framework-api}/test/json/JsonComparator.html[`JsonComparator`].
+
+Alternatively, you can use AssertJ https://github.com/assertj/assertj/blob/3.x/assertj-core/src/main/java/org/assertj/core/api/AssertFactory.java[AssertFactory].
```

Comment:
Please rephrase that to be consistent with previous sentences. For instance.

JsonUnit is also supported via the standard `convertTo`method. I am not keen to start referring to AssertJ's Javadoc for this.